scripts/vagrant-setup-env.sh

```diff
@@ -5,6 +5,7 @@ set -eo pipefail
 echo 'Setting up correct env. variables'
 echo "export GOPATH=$GOPATH" >> "$HOME/.bash_profile"
 echo "export PATH=$PATH:$GOPATH/bin:/usr/local/go/bin" >> "$HOME/.bash_profile"
+echo "export KUBECONFIG='/var/lib/kube-spawn/default/kubeconfig'" >> "$HOME/.bash_profile"
```

Comment:
nit: we don't need to use single quotes (`'`), do we?

Reply:
Have pushed a commit to remove the quotes, but doesn't show because Github is facing issue. In case you were about to merge this.